### ISSUE TITLE
Yo generator is generating code the uses connectWithDetails (an undocumented API)

### ISSUE THREAD

**Issue by davidkel** [hyperledger/composer#2685]
From RC, using 0.15.0 
```
yo hyperledger-composer:businessnetwork`
```
created
```
var adminConnection = new AdminConnection({ fs: bfs_fs });
        return adminConnection.createProfile('defaultProfile', {
            type: 'embedded'
        })
        .then(function() {
            return adminConnection.connectWithDetails('defaultProfile', 'admin', 'Xurw3yU9zI0l');
        })
```
This was done so it was working for the 0.15.0 release but needs to move to a card based impl.

**Comment by davidkel** [comment]
Hi @bestbeforetoday , assigned this to you as I think it was what we were discussing.

**Comment by bestbeforetoday** [comment]
Resolved by #2723
